Playwright.Dockerfile

```diff
@@ -1,17 +1,24 @@
-FROM mcr.microsoft.com/playwright:v1.55.0-jammy
+FROM mcr.microsoft.com/playwright:v1.57.0-jammy
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 RUN apt-get update && apt-get install -y --no-install-recommends --fix-missing \
-    python3=3.10.6-1~22.04.1 \
-    make=4.3-4.1build1 \
+    bash=5.1-6ubuntu1.1 \
+    curl=7.81.0-1ubuntu1.21 \
     g++=4:11.2.0-1ubuntu1 \
-    curl=7.81.0-* \
-    && npm install -g pnpm@10.6.5 \
+    make=4.3-4.1build1 \
+    python3=3.10.6-1~22.04.1 \
+    unzip=6.0-26ubuntu3.2 \
+    && curl --retry 5 --retry-delay 2 -fsSL https://bun.sh/install | bash -s "bun-v1.3.5" \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
 
+ENV BUN_INSTALL=/root/.bun
+ENV PATH="/root/.bun/bin:$PATH"
+
 WORKDIR /app
 
-COPY package.json pnpm-lock.yaml ./
+COPY package.json bun.lock* ./
 
-RUN pnpm install
+RUN bun install && bun x playwright install --with-deps
```

Comment:
We shouldn't install manually the bun dependencies inside dockerfiles
It should be added to the package.json and then bun install should install all dependencies that we need
Please refactor this line